R/model.R

```diff
@@ -713,6 +713,7 @@ sample <- function(data = NULL,
                    fixed_param = FALSE,
                    validate_csv = TRUE,
                    show_messages = TRUE,
+                   opencl_ids = NULL,
```

Comment:
Do you like having this as the last argument or should we put it near the arguments like parallel_chains, chain_ids, threads_per_chain, etc? I'm not sure what makes the most sense.

Reply:
Good call, will move it alongside thread_per_chain.